src/components/Page/DefaultSEO/index.tsx

```diff
@@ -42,11 +48,23 @@ const DefaultSEO: React.FC<IDefaultSEOProps> = ({ pageInfo }) => {
     },
     {
       property: 'og:image',
-      content: metaImage
+      content: metaImage.src
+    },
+    {
+      property: 'og:image:width',
+      content: metaImage.width
+    },
+    {
+      property: 'og:image:height',
+      content: metaImage.height
     },
     {
       property: 'og:image:secure_url',
-      content: metaImage
+      content: metaImage.src
+    },
+    {
+      name: 'twitter:site',
+      content: '@dvcORG'
```

Comment:
should it be actually just dvc.org URL?

Reply:
I didn't add this, but I verified it-
Despite what the field's name implies, this is how `twitter:site` supposed to be used according to Twitter.
https://developer.twitter.com/en/docs/tweets/optimize-with-cards/guides/getting-started

Reply:
yep, make sense! 👍